modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -103,9 +342,36 @@ object ElasticQuery {
       caseInsensitive = None
     )
 
+  /**
+   * Constructs a type-safe instance of [[TermQuery]] using the specified parameters. [[TermQuery]] is used for matching
+   * documents that contain an exact term in a provided field.
+   *
+   * @param field
+   *   the field for which query is specified for
+   * @param value
+   *   the value that will be used for the query, represented by an instance of type `A`
+   * @tparam A
+   *   the type of value that will be used for the query. A JSON decoder must be in scope for this type
+   * @return
+   *   an instance of [[TermQuery]] that represents the term query to be performed.
+   */
   final def term[A: ElasticPrimitive](field: String, value: A): Term[Any, A] =
     Term(field = field, value = value, boost = None, caseInsensitive = None)
 
+  /**
+   * Constructs a type-safe instance of [[WildcardQuery]] using the specified parameters. [[WildcardQuery]] is used for
+   * matching documents containing a value that contains, starts, or ends with the specified value in the specified
```

Comment:
Maybe "that matches a provided pattern value". I think the Wildcard query works as a regex.